### DISCUSSION TITLE
Moving around zoomed image

### DISCUSSION THREAD

**Discussion by johnlewisdt** [nolimits4web/swiper#6687] [Q&A]
Hi, I have a Swiper zoom and it's working great with

```
swiper.zoom.toggle(); 
```

However, zoomed in images are fixed in the centre and cannot be navigated around when the mouse pointer is moved. What would cause this? The image properties? Or do I need to override and use a secondary target image for this?

My image css is

```
#swiperImageGallery .swiper-slide img {
    height: auto;
    max-height: 95vh;
    width: auto;
    margin: auto;
}
```

My slide DOM structure is

`div.swiper-slide > div.swiper-zoom-container > img
`
Any pointers (no pun intended) are appreciated!

**Comment by webdiva69** [comment]
We are looking for the same functionality.  Following this thread. Currently you have to have the mouse clicked to move we want it just to move/pan on hover.

**Comment by broox** [comment]
I also would like this feature. Requested here: https://github.com/nolimits4web/swiper/issues/7306
